launch_ros/launch_ros/actions/node.py

```diff
@@ -362,6 +362,15 @@ def node_name(self):
             raise RuntimeError("cannot access 'node_name' before executing action")
         return self.__final_node_name
 
+    @property
+    def node_autostart(self):
+        """Getter for autostart."""
+        return False
+
+    @property
+    def is_lifecycle_node(self):
+        return False
+
```

Comment:
I don't think this should be added to `Node` either. It's a bad smell for lifecycle to be mentioned in the Node class at all. If we wanted to add another kind of node in the future we don't want to have to change the node class each time. Can we find a way to not need these two functions on node? This seems like a job for the type system.

Reply:
Done :+1: